src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -192,6 +192,10 @@
     "DESCRIPTION": "Draw !M! cards. NL This turn, your next attack is played !stsjorbsmod:MetaMagic! additional time.",
     "UPGRADE_DESCRIPTION": "Draw !M! cards. NL This turn, your next attack is played !stsjorbsmod:MetaMagic! additional times."
   },
+  "stsjorbsmod:Haunt": {
+    "NAME": "Haunt",
+    "DESCRIPTION": "Gain !stsjorbsmod:UrMagic! Intangible. NL Next turn, Draw !M! cards. NL Exhaust."
```

Comment:
Probably the most consistent wording for the draw effect would be to match Predator, something like `Next turn, draw !M! additional cards.` (lowercase draw, added word "additional")